types/tx/module/module.go

```diff
@@ -0,0 +1,104 @@
+package tx
```

Comment:
I, personally, don't like where this file is. I just doesn't feel right to have function that mutates baseapp under `types/`.

Reply:
could be in auth/tx instead?

Reply:
Yes, this makes way more sense!